packages/central-server/src/hooks/entityAttribute.js

```diff
@@ -0,0 +1,20 @@
+const snakeCased = name =>
```

Comment:
Nice little util, but can avoid adding more code by using
```suggestion
import { snake } from 'case';
```